app/src/main/res/layout-land/question_player_fragment.xml

```diff
@@ -190,4 +191,4 @@
         android:layout_height="124dp" />
     </FrameLayout>
   </FrameLayout>
-</layout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
added

Reply:
> @vrajdesai78 Firstly you cannot directly use `component_color_shared_progress_bar_stroke_color`, its not a progress bar its a congratulations text view.
> 
> So you will need to update value of `component_color_shared_correct_text_color ` in colors file.
> 
> Secondly, please share before and after screenshot side-by-side to clearly show the difference.

Added screenshot to compare before and after, also updated color_pallete.xml